test/core/end2end/grpc_core_end2end_test.bzl

```diff
@@ -73,7 +73,100 @@ _DEPS = [
     "//test/core/test_util:test_lb_policies",
 ]
 
-def grpc_core_end2end_test(name, shard_count = 1, enable_fuzzing = True, tags = [], flaky = False):
+_TESTS = [
```

Comment:
Instead of having to maintain this list, could we just glob the files in the tests directory?  Seems like it would be one less thing to maintain.

Reply:
There's some best practices around build performance that recommend against globs that I'd like to uphold wherever possible.